src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionFlag.java

```diff
@@ -0,0 +1,118 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.protection.flags.*;
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+import org.bukkit.event.Event;
+import org.skriptlang.skriptworldguard.SkriptWorldGuard;
+import org.skriptlang.skriptworldguard.worldguard.WorldGuardRegion;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExprRegionFlag extends PropertyExpression<WorldGuardRegion, Flag> {
+
+    static {
+        Skript.registerExpression(ExprRegionFlag.class, Flag.class, ExpressionType.COMBINED,"[worldguard] flag[s] %strings%", "worldguardregions");
+    }
+
+    WorldGuard wg = WorldGuard.getInstance();

```

Comment:
```suggestion
    private static final WorldGuard WORLD_GUARD = WorldGuard.getInstance();
```
This can go above the static block too.